### DISCUSSION TITLE
[postponed][x] Some samples are "detached" in Blueprint repo

### DISCUSSION THREAD

**Discussion by dtitov** [fairtracks/fairtracks_standard#114] [Team Posts]
@dzerbino, could you, please, take a look at this issue: https://github.com/elixir-no-nels/trackfind/issues/81

Basically, several samples in the Blueprint dump are not referenced from anything, which doesn't really break anything, but makes this hub be a bit decoupled...

**Comment by dzerbino** [comment]
Corrected version sent by e-mail

**Comment by sveinugu** [comment]
But there are tracks for those samples. E.g. "DG-75 H3K4me3 MACS2_wiggler Sporadic Burkitt lymphoma signal from NCMLS" has this bigdataurl: "http://ftp.ebi.ac.uk/pub/databases/blueprint/data/homo_sapiens/GRCh38/Cell_Line/DG-75/Sporadic_Burkitt_lymphoma/ChIP-Seq/NCMLS/DG-75_c01.ERX297407.H3K4me3.bwa.GRCh38.20150528.bw". Now they are just removed?

**Comment by dzerbino** [comment]
You're right, it's evidently in the track hub but not in the tracking DB. Since I need metadata from the tracking DB to link to sample etc, these get dropped.

**Comment by sveinugu** [comment]
Linked to in https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/49. Removing pin
